pr_toolbox/quantum/results/frequencies.py

```diff
@@ -14,13 +14,35 @@
 
 from collections import defaultdict
 from collections.abc import Callable
+from typing import Union
 
-from qiskit.result import Counts
+from numpy import sqrt
+from qiskit.result import Counts, QuasiDistribution
+
+FrequenciesLike = Union[Counts, QuasiDistribution]
 
 
 ################################################################################
-## COUNTS
+## FREQUENCIES
 ################################################################################
+def map_frequencies(frequencies: FrequenciesLike, mapper: Callable) -> FrequenciesLike:
+    """Map frequencies by reassigning keys according to input callable.
+
+    Args:
+        frequencies: the frequencies to process.
+        mapper: the callable to map readout bits (i.e. counts keys).
+
+    Returns:
+        New frequencies with readout bits mapped according to input callable.
+    """
+
+    if isinstance(frequencies, Counts):
+        return map_counts(frequencies, mapper)
+    if isinstance(frequencies, QuasiDistribution):
+        return map_quasi_dists(frequencies, mapper)
+    raise TypeError("Frequencies of type (Counts, QuasiDistribution) are expected for mapping.")
```

Comment:
Maybe it is good to avoid the word _'mapping'_ in the error message since functions building on top do not use it (e.g. `bitflip`). 
```suggestion
    raise TypeError(f"Invalid frequencies type. Expected `Counts` or `QuasiDistribution` but got {type(frequencies)} instead.")
```